R/JointModel.R

```diff
@@ -142,8 +142,11 @@ as.character.JointModel <- function(x, ...) {
 
 #' @rdname write_stan
 #' @export
-write_stan.JointModel <- function(object, file_path) {
-    fi <- file(file_path, open = "w")
+write_stan.JointModel <- function(object, destination) {
+    if (is_connection(destination)) {
+        return(writeLines(as.character(object), fi))
```

Comment:
Something like this, right?
```suggestion
        return(writeLines(as.character(object), con = destination))
```